### ISSUE TITLE
Zoom in shortcut icons and trackpad zooming don't seem to work 

### ISSUE THREAD

**Issue by dickinson0718** [dulimarta/spherical-easel#82]
If you activate the zoom in/out/fit tool from the side menu (the zoom tools under "Display Tool"), the tools and zooming works fine But if you click the zoom in button next to the message center it doesn't seem to work. (Zoom fit and zoom out does work.)

Zooming with the trackpad, enlarges the label font size and the line thickness, when it shouldn't.

**Comment by dulimarta** [comment]
While trying to resolve this issue, I noticed that we have two "global events" related to zooming.

1. `magnification-updated`
   * fired by `stores/se.ts` in function `setZoomMagnificationFactor`
   * listened to by `Easel.vue::resizePlottables()`
2. `zoom-updated`
   * fired by `commands/ZoomSphereCommand.ts` and `eventHandlers/PanZoomHandler.ts`
   * listened to by `SphereFrame.vue::updateView()`

In both `ZoomSphereCommand` and `PanZoomHandler.ts` we have the following snippet:

```
Command.store.setZoomMagnificationFactor(___)
EventBus.fire("zoom-updated")
```

The 2-line snippet causes both `magnification-updated` and `zoom-updated` to be triggered.

I'm thinking of cleaning up this redundancy.

**Comment by dulimarta** [comment]
The `clickFunc` function provided in `tooldictionary.ts` does not run properly when it is passed into `ShortcutIcon.vue`

**Comment by dulimarta** [comment]
Fixed in commit 61580cdfc88d8, except for the "Zoom Fit" shortcut. The zoom fit requires canvas dimension which is not easily accessible in `tooldictionary.ts`

**Comment by dulimarta** [comment]
The Zoom Fit shortcut icon is now functional again in commit c1fa5b3.
